artifacts/flagger/crd.yaml

```diff
@@ -82,6 +82,8 @@ spec:
                 interval:
                   type: string
                   pattern: "^[0-9]+(m|s)"
+                iterations:
+                  type: number
```

Comment:
Don't we have to include match field in the crd as well?

Reply:
We don't have validation for any of the Istio types (`HTTPMatchRequest`, `HTTPRewrite`, `HTTPRetry`, `Headers` and `CorsPolicy`) used in the Canary CRD. This could be addressed in a separate PR.

Reply:
sounds good